### PR TITLE
annotation examples

### PR DESCRIPTION
**Pull request by msokoloff1**
* Add examples for using annotation types
* Made changes to core labelbox to use annotation types by default on label export.

### CODE REVIEW

File: labelbox/schema/ontology.py [hunk 1]
```diff
@@ -4,11 +4,13 @@
 
 from typing import Any, Dict, List, Optional, Union
 
-from labelbox.schema.project import Project
+#from labelbox.schema.project import Project
 from labelbox.orm.db_object import DbObject
 from labelbox.orm.model import Field, Relationship
 from labelbox.exceptions import InconsistentOntologyException
 
+Project = "NONE"
```
Comment (awu-labelbox) [inline labelbox/schema/ontology.py:12]: Were these file's changes for debug?

File: labelbox/schema/project.py [hunk 2]
```diff
@@ -166,7 +167,14 @@ def export_queued_data_rows(self, timeout_seconds=120):
                 self.uid)
             time.sleep(sleep_time)
 
-    def export_labels(self, timeout_seconds=60):
+    def export_labels(self):
+        json_data = self.export_labels_json()
+        if 'frames' in json_data[0]['Label']:
+            # Assumes all data rows in this project are video.
```
Comment (awu-labelbox) [inline labelbox/schema/project.py:173]: This seems like a stretch of an assumption. Maybe add a docstring that mentions this?

File: labelbox/schema/ontology.py (hunk 1 above)
Comment (msokoloff1) [reply labelbox/schema/ontology.py:12]: Yeah this needs to be fixed. good catch

File: labelbox/schema/project.py (hunk 2 above)
Comment (msokoloff1) [reply labelbox/schema/project.py:173]: Yeah I think I am going to create a separate function for exporting video.

**Review by awu-labelbox — APPROVED** [review]

**Pull request merged**